services/administration/src/EShopOnAbp.AdministrationService.EntityFrameworkCore/EntityFrameworkCore/AdministrationServiceDbContext.cs

```diff
@@ -52,7 +47,6 @@ protected override void OnModelCreating(ModelBuilder builder)
 
             builder.ConfigurePermissionManagement();
             builder.ConfigureSettingManagement();
-            builder.ConfigureFeatureManagement();
```

Comment:
I suppose that we should add a new migration to drop the feature management tables from the database.

Reply:
You are right. Created a new migration for AdministrationService.